script/app-mlperf-inference-nvidia/customize.py

```diff
@@ -641,15 +641,15 @@ def preprocess(i):
             run_config += f" --audio_buffer_num_lines={audio_buffer_num_lines}"
 
         use_fp8 = str(env.get('MLC_MLPERF_NVIDIA_HARNESS_USE_FP8', ''))
-        if use_fp8 and use_fp8.lower() not in ["no", "false", "0", ""]:
+        if use_fp8 and is_true(use_fp8):
```

Comment:
Not is_false is not the same as is_true 🙂. That's the case for boolean values only. If a variable is unset, the behaviour is changing here.